firmware/Core/Src/timekeeping/timekeeping.c

```diff
@@ -8,24 +8,21 @@
 uint64_t TIM_unix_epoch_time_at_last_time_resync_ms = 0;
 uint32_t TIM_system_uptime_at_last_time_resync_ms = 0;
 TIM_sync_source_t TIM_last_synchroization_source = TIM_SOURCE_NONE;
-__IO uint64_t uwTick64 = 0;
-
-extern __IO uint32_t uwTick;
 
 uint64_t TIM_get_current_system_uptime_ms(void) {
```

Comment:
I suggest returning a `uint32_t` from TIM_get_current_system_uptime_ms()`, though I don't see it making a huge difference.

Reply:
Thanks for pointing that out!

Reply:
Branch updated and pushed